knowledge-base/kb-pdfviewer-argumentnullexception-pages.md

```diff
@@ -0,0 +1,231 @@
+﻿---
+title: "Value cannot be null. Parameter name: Pages" exception opening a particular pdf
+description: This article explains the reasons for receiving an ArgumentNullException with message "Value cannot be null. Parameter name: Pages" when trying to show PDF document.
+type: troubleshooting
+page_title: ArgumentNullException when importing document in PdfViewer
+slug: kb-pdfviewer-argumentnullexception-pages
+position: 
+tags: 
+ticketid: 1401132
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>RadPDFViewer for WPF</td>
+	</tr>
+</table>
+
```

Comment:
You can remove this empty line. It comes from the templated .md file and annoys me a lot. :D